config/application.rb

```diff
@@ -51,11 +51,14 @@ class Application < Rails::Application
     end
 
     config.generators do |g|
-      g.factory_bot true
-      g.helper false
-      g.javascripts false
-      g.stylesheets false
+      g.factory_bot nil
```

Comment:
Can we get the factory_bot to auto generate? I think that is the existing behavior, which I think is nice.